NEW_RELEASE_NOTES.md

```diff
@@ -10,3 +10,4 @@ appropriate header in [RELEASE_NOTES.md](./RELEASE_NOTES.md).
 - materials: fix alpha masked materials when MSAA is turned on [⚠️ **Recompile materials**]
 - materials: better support materials with custom depth [**Recompile Materials**]
 - engine: fade shadows at shadowFar distance instead of hard cutoff [⚠️ **New Material Version**]
+- engine: fix critical regression where enabling screen-space reflections would crash
```

Comment:
No release note needed since the problematic change ("Remove now-redundant feature level 0 materials") has not been released yet.